Makefile

```diff
@@ -206,8 +201,10 @@ upload-prerelease-container-image: release
 	docker build -t gcr.io/gadget-core-production/dateilager:$(GIT_COMMIT) .
 	docker push gcr.io/gadget-core-production/dateilager:$(GIT_COMMIT)
```

Comment:
I suspect that if we called this upload function (which should ultimately live in a Github Actions) it will push an ARM image to our registry.

Reply:
You're right! The whole reason I was messing around with this is to be able to build a local image for testing locally, but also push a prerelease image to the real registry for use in Gadget's CI to use as an e2e test of the whole thing. I had multiarch builds set up in the downstream branch but didn't git-fu it up here in this one, moved it now so its its final form on this branch